isogram/isogram.py

```diff
@@ -1,2 +1,7 @@
-def is_isogram(string):
-    pass
+def is_isogram(word):
+    if isinstance(word,str):
+        w = word.lower() # assuming you want a case in-sensitive search
+        return word, len(w) == len(set(w)) if w else False
```

Comment:
This is supposed to check that the word doesn't repeat any letters, I do not see you addressing this.

Reply:
I used the set() to convert the string to a "set", which removes the duplicate letters (set doesn't allow duplicate elements), then compare the length.